### ISSUE TITLE
Showing black screen at reload and app open

### ISSUE THREAD

**Issue by ebelevics** [lulupointu/vrouter#121]
I found weird behavior on app, where when I reload the app or close it and reopen it, it always starts transition animation from black screen. I couldn't tell why at first but then I found that Futures on main() causes it, on normal MaterialApp I don't have these problems. How can I set app open animation to normal behavior. 

```
void main() async {
  WidgetsFlutterBinding.ensureInitialized();
  await Future.delayed(const Duration(milliseconds: 500));
  runApp(MainApp());
}

class MainApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return VRouter(
      routes: [
        VWidget(
          path: '/',
          widget: LoginWidget(),
        ),
      ],
    );
  }
}

class LoginWidget extends StatelessWidget {
  const LoginWidget({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Container(
      color: Colors.white,
      child: Center(child: Text("Test Page")),
    );
  }
}
```

I have added small video fragment to see better behavior https://files.fm/u/ef7me6agc

**Comment by lulupointu** [comment]
I understand what you need and I can reproduce this. 

I have found a solution but I want to fix #116 in the next update so you will have to wait a few days.
I'll update you when the update comes out and close this issue then.

PS: Thanks for the nicely written issue

**Comment by ebelevics** [comment]
Ok, thanks. Then I can close it for now.

**Comment by lulupointu** [comment]
This should be fixed with `v1.2.0+6`. Could you check and tell me if it works has you expect (because I'm not 100% what I fixed was your issue 😅).

**Comment by ebelevics** [comment]
The problem is fixed. :)
